vendor.conf

```diff
@@ -1,23 +1,25 @@
 github.com/urfave/cli v1.17.0
-github.com/containers/image master
+github.com/containers/image issue-327 https://github.com/hferentschik/image.git
```

Comment:
can you vendor this from `master` again?

Reply:
Sorry, that was my intention